config/adopters.json

```diff
@@ -313,6 +313,15 @@
 				"iot.zenoh"
 			]
 		},
+		{
+			"name": "flx:business",
+			"homepage_url": "https://flx-business.de/",
```

Comment:
The website is not available, showing "503 Service Unavailable" to me.
It's not our fault, but might be better to check with them if they offered a correct URL?